pages/api/update-email-by-id.tsx

```diff
@@ -0,0 +1,56 @@
+import { verifyAdminToken } from "@/lib/auth";
+import { FirebaseTablesEnum } from "@/lib/enums";
+import { initializeAdmin } from "@/lib/firebase-admin";
+import * as admin from "firebase-admin";
+
+async function updateEmailById(
```

Comment:
Curious why this is separate from the general update member endpoint...It may make it easier to just abstract the details on the backend rather than create 2 separate endpoints.

Reply:
This one's a bit tricky. I'm not including this in the general update member is because it's an entirely separate table - `SECURE_MEMBER_DATA` meaning it's not updating any fields for the member.

For context, we made this to keep sensitive member data separated so that in the future public member data could be developed without the need to set specific rules on the member table fields

I'm leaning towards keeping this endpoint separated since this email field doesn't actually exist in the members table, but lmk what you think

For now, to make sure it's clear, I renamed the file to `update-secure-email-by-id`